### DISCUSSION TITLE
Install on IIS & windows server

### DISCUSSION THREAD

**Discussion by sumoo13** [ltb-project/self-service-password#765] [Q&A]
Hi everybody,

I already have an instance installed on a debian server, but for internal purposes I would like to test it on a Windows server with IIS.

I have the error: PHP Fatal error:  Uncaught Error: Failed opening required '/usr/share/php/smarty3/Smarty.class.php'
apparently it will search directly on the path /usr/share/

is it possible to do otherwise?

**Comment by coudot** [comment]
Configure SMARTY parameter in config.inc.php

**Reply by sumoo13** [reply]
Thank you so much
I had not pushed the exploration of the config.inc.php file
only the local
